lib/rules/no-multi-comp.js

```diff
@@ -61,21 +103,46 @@ module.exports = {
       );
     }
 
-    return {
+    /**
+     * Checks if the component is exported, if exportOnly is set
+     * @param {Object} component The component being checked.
+     * @returns {boolean} True if the component is exported or exportOnly is false
+     */
+    function isPrivate(component) {
+      return ignorePrivate && exportedComponents.has(findComponentIdentifierFromComponent(component));
+    }
+
+    const rule = {
       'Program:exit'() {
         if (components.length() <= 1) {
           return;
         }
 
         values(components.list())
           .filter((component) => !isIgnored(component))
```

Comment:
```suggestion
```

Reply:
ah yeah, the commit suggestion button resulted in some unhelpful commits. I'll resolve these manually.